lib/FiniteStateEntropy/README.md

```diff
@@ -1,9 +1,24 @@
+**Note:** This static version is a fork of https://github.com/Cyan4973/FiniteStateEntropy/commit/510d22b221c8c02f281c35f9edeb606baacef27b with the following changes made to fse.h
+
+```
+662c662
+< #  define FSE_MAX_MEMORY_USAGE 16
+---
+> #  define FSE_MAX_MEMORY_USAGE 14
+665c665
+< #  define FSE_DEFAULT_MEMORY_USAGE 16
+---
+> #  define FSE_DEFAULT_MEMORY_USAGE 13
+```
+
+An alternate CMake fetch conten based CMakeLists.txt for the chiapos project is included as fetch-content-CMakeLists.txt should we revisit this fork strategy.
```

Comment:
```suggestion
An alternate CMake fetch content based CMakeLists.txt for the chiapos project is included as fetch-content-CMakeLists.txt should we revisit this fork strategy.
```

Reply:
Fixed and added another line to explain the state of the cmakelist.txt...